app/src/main/DbAdaptersAndroidManifest.xml

```diff
@@ -0,0 +1,5 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
File name: prefer spelling out the thing rather than abbreviating. Ditto for the next manifest.

Reply:
Done!